feature/mypage/src/main/java/com/terning/feature/mypage/mypage/MyPageRoute.kt

```diff
@@ -116,7 +120,7 @@ fun MyPageRoute(
         }
```

Comment:
113부터 119가 예전 코드라 태그가 안되네요;;
LaunchedEffect와 DisposableEffect를 합쳐도 되지 않을까 싶네요!
```suggestion
      DisposableEffect(lifecycleOwner) {
            systemUiController.setStatusBarColor(color = Back)
            onDispose {
                  systemUiController.setStatusBarColor(color = White)
            }
      }
```

Reply:
오호 혹시 어떻게 합친다는 의미인지 여쭤봐도 될까용...??

`LaunchedEffect`는 컴포지션이 생성될 때, `DisposableEffect`는 컴포지션이 사라질 때 적용된다고 생각해서 마이페이지 전용 상태바 색인 `Back`을 넣구 다른 화면으로 갈 때는 원래 상태바 색인 `White`로 돌아가게 했었어요..!

Reply:
아! 코드를 제시해주셨군요!! 😅😅 반영해놓겠습니다~ 공부해올게요.